### ISSUE TITLE
[Accessibility]: Chart toolbar buttons don't have tooltips

### ISSUE THREAD

**Issue by nemeseri-ibm** [carbon-design-system/carbon-charts#1549]
### Name

Andras Nemeseri

### Are you an IBM employee?

- [X] Yes

### Environment

Browser

### What happened? What did you expect to happen instead?

**What happened?**
The chart toolbar buttons do not have tooltips describing the actions they do.

**What did you expect to happen instead?**
The buttons should show tooltips describing the actions they do to provide better usability for the users.

### What WCAG 2.1 checkpoint does the issue violate?

none

### Version

@carbon/charts@1.6.14

### What priority level would this issue be in your opinion?

P0

**Comment by theiliad** [comment]
we'd probably add the `title` prop here
